exporter/fileexporter/README.md

```diff
@@ -21,16 +20,24 @@ This intended for primarily for debugging Collector without setting up backends.
 ## Getting Started
 
 The following settings are required:
+
 - `path` (no default): where to write information.
 
 The following settings are optional:
+
 - `rotation` settings to rotate telemetry files.
-    - max_megabytes:  [default: 100]: the maximum size in megabytes of the telemetry file before it is rotated.
-    - max_days: [no default (unlimited)]: the maximum number of days to retain telemetry files based on the timestamp encoded in their filename.
-    - max_backups: [default: 100]: the maximum number of old telemetry files to retain.
-    - localtime : [default: false (use UTC)] whether or not the timestamps in backup files is formatted according to the host's local time.
+
+  - max_megabytes:  [default: 100]: the maximum size in megabytes of the telemetry file before it is rotated.
+  - max_days: [no default (unlimited)]: the maximum number of days to retain telemetry files based on the timestamp encoded in their filename.
+  - max_backups: [default: 100]: the maximum number of old telemetry files to retain.
+  - localtime : [default: false (use UTC)] whether or not the timestamps in backup files is formatted according to the host's local time.
+
+- `marshaler`[default: json] :the type of marshaler which encodes telemetry data. The setting can be overridden with `proto` or `protobuf`.
```

Comment:
I can appreciate what you are doing with allowing both `proto` and `protobuf`, but generally in this project we find it better to just have one value for given functionality. This keeps things clear for the user and helps us simplify the code.